COMMUNICATON.md

```diff
@@ -0,0 +1,33 @@
+# Communication
+
+## Code of conduct
+
+This project has adopted the [GitHub community guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines).
+
+## Issues
+
+Issues are used as the primary method for tracking anything to do with the Fairlearn project.
```

Comment:
are users supposed to ask usage questions through issues as well?

Reply:
That's been happening already. My feeling right now is "yes" although there are probably other forums for such questions as well, e.g. StackOverflow. [note: I am monitoring StackOverflow, but there are no questions yet AFAIK].

Any idea how other projects handle usage questions? I'd love to be as specific with this as possible, in order to ensure that people know how to reach out.

Reply:
in sklearn if you try to create an issue as a usage question, you'll see:

> **If your issue is a usage question, please submit it in one of these other channels instead:**
> - **StackOverflow with the scikit-learn tag: https://stackoverflow.com/questions/tagged/scikit-learn**
> - **Mailing List: https://mail.python.org/mailman/listinfo/scikit-learn**
> - **Gitter: https://gitter.im/scikit-learn/scikit-learn**
> - **For more information, see User Questions: http://scikit-learn.org/stable/support.html#user-question**
> 
> The issue tracker is used only to report issues and feature requests. For
> questions, please use either of the above platforms. Most question issues are
> closed without an answer on this issue tracker. Thanks for your understanding.
>

Reply:
How do these automatically come up? Any idea?

Reply:
I think I've discovered this in GitHub settings. I'll see what I can do about this.

Reply:
This is part of the ymls now